ddl/column.go

```diff
@@ -121,42 +121,43 @@ func createColumnInfo(tblInfo *model.TableInfo, colInfo *model.ColumnInfo, pos *
 	return colInfo, position, nil
 }
 
-func onAddColumn(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, err error) {
-	// Handle the rolling back job.
-	if job.IsRollingback() {
-		ver, err = onDropColumn(t, job)
```

Comment:
Could we call `onDropColumn` in `rollingbackAddColumn` directly? Then we needn't write the logic of `onDropColumn` again.